ROOTFS/etc/waggle/sanity/fatal/core_rpi.test

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+pf="RPI test"
+echo "$pf: Begin"
+
+# check core for rpi mount first
+if [ "$(df -h | grep "/media/rpi")" = "" ];
+then 
+        echo "$pf: RPI not mounted at /media/rpi FAIL"
+        exit 9
+fi
+
+# make sure nfs-server is running on core
+if [ "$(systemctl status nfs-server.service | grep "active (exited)")" = "" ];
+then
+        echo "$pf: nfs-server not running on core FAIL"
+        exit 10
+fi
+
+if [ "$(systemctl status dnsmasq.service | grep "active (running)")" = "" ]; then
+        echo "$pf: dnsmasq is not running FAIL"
+        exit 11
+fi
+
+# historial fsck
+uptime_timestamp=$(uptime --since)
+
+if [ "$(journalctl --since "$uptime_timestamp" | grep "/dev/mmcblk0p11: clean")" = "" ];
+then
+        echo "$pf: dnsmasq did not finish sending files to RPI FAIL"
+        exit 12
+fi
+
+if [ ! -f /etc/waggle/node_manifest.json ];
+then
+        echo "$pf: node_manifest.json does not exist FAIL"
+        exit 13
+fi
+
+if cat /etc/waggle/node_manifest.json | jq .shield.present | grep -q true; then 
+
+        # testing if we can log into the RPI. might remove later for redundancy
+        if [ "$(ssh rpi "echo ping")" != "ping" ];
```

Comment:
I like this test specifically, will let us know the difference between not being able to ssh into the rpi and the RO partition not being mounted. Please don't remove

Reply:
I prefer a string other then "ping" as there exists a `ping` command and I was a little confused what you were doing.